modules/admin-ui-index/src/main/resources/OSGI-INF/message-receiver-workflow-definition.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0"
+               name="org.opencastproject.index.service.message.WorkflowDefinitionMessageReceiverImpl-Adminui"
+               immediate="true">
+  <implementation
+    class="org.opencastproject.index.service.message.WorkflowDefinitionMessageReceiverImpl"/>
+  <property name="service.description" value="Workflow Definition Message Receiver"/>
+  <property name="destinationId" value="WORKFLOW_DEFINITION.Adminui"/>
+  <service>
+    <provide interface="org.opencastproject.index.service.message.WorkflowDefinitionMessageReceiverImpl"/>
+  </service>
+
+  <reference name="message-broker-sender"
+             interface="org.opencastproject.message.broker.api.MessageSender"
+             bind="setMessageSender"/>
+
+  <reference name="message-broker-receiver"
+             interface="org.opencastproject.message.broker.api.MessageReceiver"
+             bind="setMessageReceiver"/>
+
+  <reference name="message-receiver-lock-service"
+             interface="org.opencastproject.index.service.message.MessageReceiverLockService"
+             bind="setMessageReceiverLockService"/>
+
+  <reference name="search-index"
+             interface="org.opencastproject.adminui.index.AdminUISearchIndex"
+             bind="setSearchIndex"/>
+
+  <reference name="security-service"
+             interface="org.opencastproject.security.api.SecurityService"
+             bind="setSecurityService"/>
+
+</scr:component>
```

Comment:
I don't quite understand how there are two message receivers with only one implementation, could you elaborate how this works?